src/components/DataPage/DataPage.svelte

```diff
@@ -87,6 +87,7 @@
 		checkboxFilterList.map((group) => [group.key, group.choices.map((choice) => choice.value)])
 	);
 	export let mounted = false;
+	export let donwloadSize = 'sm';
```

Comment:
Typo : `downloadSize` + add type of possible value
```ts
export let downloadSize: 'sm' | 'lg' | 'otherPossibleValue' = 'sm';
```

Reply:
[b0572](https://github.com/wevisdemo/parliament-watch/pull/71/commits/b0572f73f277c0365b6274ef744f085a74c1dff6)